src/v2/kuadrant/resolver/wasm_host.rs

```diff
@@ -30,4 +30,22 @@ impl AttributeResolver for ProxyWasmHost {
             ))),
         }
     }
+
+    fn set_attribute_map(
+        &self,
+        map_type: proxy_wasm::types::MapType,
+        value: HashMap<String, String>,
+    ) -> Result<(), AttributeError> {
+        //TODO: review the abomination of converting to Vec<(&str, &str)>
```

Comment:
Yep we discussed and I agree, will think about this as a follow up and how these are best stored